src/settings.rs

```diff
@@ -0,0 +1,40 @@
+use serde::{Deserialize, Serialize};
+use std::collections::HashSet;
+
+// Describe the settings your policy expects when
+// loaded by the policy server.
+#[derive(Serialize, Deserialize, Default, Debug)]
+#[serde(default)]
+pub(crate) struct Settings {
+    #[serde(default)]
+    pub allowed_profiles: HashSet<String>,
+    pub profile_types: HashSet<String>,
+    pub localhost_profiles: HashSet<String>,
+}
+
+impl kubewarden::settings::Validatable for Settings {
+    fn validate(&self) -> Result<(), String> {
+        let allowed_profile_types: HashSet<_> = [
```

Comment:
personal taste, you can ignore that, I prefer to be explicit about the type of a collection -> `HashSet<String>`